bots/run-queue

```diff
@@ -26,11 +26,68 @@ import ssl
 import subprocess
 import sys
 import time
+import logging
+import pipes
 
 import pika
 from task import redhat_network
 
+logging.basicConfig(level=logging.INFO)
 
+# Returns a command to execute and the delivery tag needed to ack the message
+def consume_webhook_queue(channel, amqp):
+    # interpret payload
+    # call tests-scan or issue-scan appropriately
+    method_frame, header_frame, body = channel.basic_get(queue='webhook')
+    if not method_frame:
+        # No message available
+        logging.warning('No message in the webhook queue available')
```

Comment:
Let's drop this warning. The code below already has a general log message about it, and it's not an unexpected state (so it should be logging.info at most).